index.html

```diff
@@ -9,44 +9,216 @@
+              </button>
+            </nav>
+          </div>
         </header>
         <section class="hero-comtain">
           <div class="background-container">     
             <div class="contain">
                 <div>
                    <h1>Hey there. I’m Roland.<br>
                         <span>I’m a software developer</span></h1>
-                        <p> I can help you build a product , feature or website Look through some of my work and<br> experience! If you like what you see and have a project you need coded,<br> 
+                        <p class="hero-paragraph"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
                             don’t hesitate to contact me.</p>
                 </div>      
             </div>
-            <div class="icons">
-                <div><i class="fa-brands fa-github"></i></div>
-                <div><i class="fa-brands fa-linkedin"></i></div>
-                <div><i class="fa-solid fa-hand-peace"></i></div>
-                <div><i class="fa-brands fa-twitter"></i></div>
-                <div><i class="fa-sharp fa-solid fa-m"></i></div>
+            <ul class="icons">
+              <li>
+                <a href="https://github.com/RolandAmini"><i class="fa-brands fa-github"></i></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/roland-amini-b72957227/"><i class="fa-brands fa-linkedin"></i></a>
+              </li>
+              <li>
+                <a href="https://www.instagram.com/aminiroland/"><i class="fa-solid fa-hand-peace"></i></a>
+              </li>
+              <li>
+                <a href="https://twitter.com/Isaac_roland02"><i class="fa-brands fa-twitter"></i></a>
+              </li>
+              <li>
+               <a href="mailto:aminiroland01@gmail.com"><i class="fa-sharp fa-solid fa-m"></i></a>
+               </li>
     
             </div>
-        </div>     
+            <div class="hero-srolldown">
+              <a href="#">
+                <i class="fa-solid fa-chevron-down"></i>
+              </a>
+            </div>
+          </div>     
         </section>
+
+        <!---------------->
+
+
+         <section class="portfolio-section">
+    <h1 class="section-title">
+      My Recent Works
+      <hr class="section-title_line">
+    </h1>
+
+    <div class="cards">
+      <!-- card -->
+      <div class="card">
+        <img src="project-img.png" alt="project">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
```

Comment:
Great work was done so far by using semantic tags and descriptive classNames, only some small styling adjustements needed : 

1. the title needs to be bolder and the size needs to be increased
2. the cards horizontal margin needs to be increased to match the given design

![image (14)](https://github.com/RolandAmini/portfolio-mobile/assets/59707859/03e5464a-118c-4b10-b985-910ac2a94609)
![image (15)](https://github.com/RolandAmini/portfolio-mobile/assets/59707859/370c6305-cb03-4c4b-876e-acce531a791b)